Homework_2.sql

```diff
@@ -0,0 +1,33 @@
+USE baseball;
```

Comment:
Not important comment: Generally the sql file start with 
SHOW ERRORS;
SHOW WARNINGS;

Reply:
I have added this.